app-modules/engagement/src/Filament/Pages/MessageCenter.php

```diff
@@ -337,7 +339,7 @@ protected function getViewData(): array
 
             $studentPopulationQuery = Student::query()
                 ->when($this->search, function ($query, $search) {
-                    $query->where('full_name', 'like', "%{$search}%")
+                    $query->where(new Expression('lower(full_name)'), 'like', '%' . Str::lower($search) . '%')
```

Comment:
Let's actually play it a little safer here and use binding to reduce the risk of SQL injection. Something like:

```php
->whereRaw('lower(full_name) like ?', ['%' . Str::lower($search) . '%'])
```